assets/out

```diff
@@ -132,7 +133,7 @@ elif [ -n "$build" ]; then
     docker pull "${ecr_image}"
   fi
 
-  docker build -t "${repository}:${tag_name}" $expanded_build_args -f "$dockerfile" $cache_from "$build"
+  docker build -t "${repository}:${tag_name}" $expanded_build_args -f "$dockerfile" --squash "$squash" $cache_from "$build"
```

Comment:
This will always squash. It is a boolean, so if it exists then it will squash. The way it is setup here, will always squash even if it is false/true.